tools/install-promethues-server.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
```

Comment:
This file should be made executable, e.g., `chmod +x`.

Reply:
It's still not executable.

Also, **prometheus** in the filename is misspelled I believe.

Reply:
It's still not executable.

Also, **prometheus** in the filename is misspelled I believe.